### PR TITLE
Init non content

### PR DESCRIPTION
**Pull request by ostolero**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/40607

## Description
Fix a bug in Init command in SDK - When command is called from a repo that is not content we fail on error due to white-list file missing. Continue run without white list in case it doesn't exist.

### CODE REVIEW

File: demisto_sdk/commands/init/initiator.py [hunk 1]
```diff
@@ -415,10 +415,14 @@ def find_secrets(self):
...
-                f.write(secret)
-                f.write('\n')
+        try:
+            with open(f'Packs/{pack_dir}/.secrets-ignore', 'a') as f:
+                for secret in secrets:
+                    f.write(secret)
+                    f.write('\n')
+        except FileNotFoundError:
+            print_warning("Could not find the .secrets-ignore file - make sure your path is correct and is of format "
+                          "Packs/my_pack_name")
```
Comment (roysagi) [inline demisto_sdk/commands/init/initiator.py:425]: can the user really make sure? isn't this an automatic thing that happens in the init process?

**Review by roysagi — COMMENTED** [review]
See my comment, also please add a changelog entry.

File: demisto_sdk/commands/init/initiator.py (hunk 1 above)
Comment (ostolero) [reply demisto_sdk/commands/init/initiator.py:425]: Your'e right it's redundant. will remove

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,7 @@
 * Added validation to **validate** for release notes config file.
 * Added **isoversize** and **isautoswitchedtoquietmode** fields to the playbook schema.
 * Added to the **update-release-notes** command `-bc` flag to generate template for breaking changes version.
+* Added support for **init** command to run from non-content repo.
```
Comment (roysagi) [inline CHANGELOG.md:15]: move this to the changelog header please

**Review by roysagi — APPROVED** [review]
Seems good to me - just move the changelog entry please

File: CHANGELOG.md (hunk 2 above)
Comment (ostolero) [reply CHANGELOG.md]: done

**Pull request merged**